Mathlib/Algebra/Homology/ShortComplex/Refinements.lean

```diff
@@ -0,0 +1,112 @@
+/-
+Copyright (c) 2023 Joël Riou. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Joël Riou
+-/
+import Mathlib.Algebra.Homology.ShortComplex.Exact
+
+/-!
+# Refinements
```

Comment:
Would it make sense to move this file in the AbCat folder? I don't think it necessarily should live in `Homology/ShortComplex`, because it can be useful in more general settings, right?